src/main/java/seedu/duke/ui/Formatter.java

```diff
@@ -77,13 +85,10 @@ public static String formatNotes(String header, ArrayList<Note> notes) {
     public static String formatNote(String header, Note note) {
         String formattedString = "";
 
-        if (!note.getTagsName().isBlank()) {
-            header = header.concat(" " + note.getTagsName());
-        }
+        header = header.concat(" " + note.getTagsName());
```

Comment:
Realise that this + note.getTagsName can be added when this function is called. i.e. in the command execute

return formatNote(header + note.getTitle + note.getTagsName(), note)

Reply:
wouldnt it makes sense to break all of the note stuff in the formatter? and just pass the success parameter?